src/infrastructure/data-access/typeorm/user.entity.ts

```diff
@@ -14,6 +14,11 @@ export class UserModel {
   @PrimaryGeneratedColumn()
   user_id: number;
 
+  @Column({
+    name: 'username',
```

Comment:
e không validate chỗ này hả. username nhập bao nhiêu kí tự hay để trống cũng được hả e

Reply:
@dinhdung2207 use snake_case for column name in database.